test/test.js

```diff
@@ -0,0 +1,160 @@
+var run = require('comandante')
+var ipfsd = require('../index.js')
+var assert = require('assert')
+var fs = require('fs')
+var rimraf = require('rimraf')
+
+// happy lint
+// var it, before, after
```

Comment:
Does this actually work to allow standard to not complain about undefined? I usually see something like:
```
/* global beforeEach, describe, it */
```